k8s/environments/sbox/common-overlay/neuvector/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
why is this a neuvector kustomization and not an environment one?

and also we prefer to keep the kustomization's per cluster so that we can upgrade one cluster a t a time